### DISCUSSION TITLE
My email has already been used to obtain discount on GitHub account

### DISCUSSION THREAD

**Discussion by ymvs84** [community/community#144287] [GitHub Education]
### Hi everyone,

Hello.
My previous github account (Yms84), now deleted, has the student benefit.
I wanted, if possible, to move it to my new account: ymvs4.
Thank you very much.

**Comment by davevad93** [comment]
Hi @ymvs84 ,

GItHub Education benefits are tied to the academic email used to submit the application and it's not possible to transfer academic benefits from one account to another.

**Reply by ymvs84** [reply]
Hi @davevad93 I hope you are well.
Thank you for your reply.
My other account has been closed, is there any option to recover it so I can continue to have those benefits?
Or else, cancel the benefit in that account and register it in the new one.
Thank you in advance.
BR

**Reply by davevad93** [reply]
Hey @ymvs84 you're welcome , as far as I know it's not possible to remove benefits from an account. I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.
